account_invoice_comment_template/views/base_comment_template_view.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <menuitem name="Document Comments"
+    <menuitem name="Document comments"
```

Comment:
Why changing this? It follows current style of capital letters (although I don't like either).